### PR TITLE
Replace depreceted document.createEvent

### PR DESCRIPTION
**Pull request by marusak**
I kept only `StorageEvent` in `src/base1/cockpit.js` as I did not find proper documentation and I don't want to just guess.

### CODE REVIEW

File: pkg/kubernetes/scripts/kube-client-cockpit.js [hunk 1]
```diff
@@ -475,37 +475,27 @@
                             });
 
                             channel.addEventListener("close", function(ev, options) {
-                                var problem = options.problem || "";
                                 channel = null;
 
                                 state = 3;
-                                var cev = document.createEvent('Event');
-                                cev.initEvent('close', false, false, !!problem, 1000, problem);
+                                var cev = new Event("close", { "bubbles": false, "cancelable": false });
```
Comment (marusak) [inline pkg/kubernetes/scripts/kube-client-cockpit.js]: TBH here I am bit confused as in original `initEvent` takes 6 arguments but `initEvent` is specified only with 3.

**Comment by marusak** [comment]
Why no tests were performed of this? @Gundersanne @martinpitt do you have some idea?

**Comment by martinpitt** [comment]
This corresponds to this [webhook event](https://github.com/cockpit-project/cockpit/settings/hooks/31949552): from time "2019-02-15 11:36:58":

    X-GitHub-Delivery: 9f4f6ff0-310d-11e9-86f4-bd0ae2a1074a
    X-GitHub-Event: pull_request

```json
{
  "action": "opened",
  "number": 11183,
  "pull_request": {
    "url": "https://api.github.com/repos/cockpit-project/cockpit/pulls/11183",
    "id": 253381611,
    "node_id": "MDExOlB1bGxSZXF1ZXN0MjUzMzgxNjEx",
    "html_url": "https://github.com/cockpit-project/cockpit/pull/11183",
    "diff_url": "https://github.com/cockpit-project/cockpit/pull/11183.diff",
    "patch_url": "https://github.com/cockpit-project/cockpit/pull/11183.patch",
    "issue_url": "https://api.github.com/repos/cockpit-project/cockpit/issues/11183",
    "number": 11183,
    "state": "open",
    "locked": false,
    "title": "Replace depreceted document.createEvent",
    [...]
```

So the webhook event was sent.

File: pkg/kubernetes/scripts/kube-client-cockpit.js (hunk 1 above)
Comment (martinpitt) [reply pkg/kubernetes/scripts/kube-client-cockpit.js]: That looks/looked like an ancient copy&paste error. The [documentation](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent#Syntax) says `initEvent()` only takes three arguments, so the others were most probably simply ignored and we just never noticed.

File: pkg/kubernetes/scripts/graphs.js [hunk 2]
```diff
@@ -541,8 +541,11 @@
                                 }
                             }
 
-                            var event = document.createEvent("CustomEvent");
-                            event.initCustomEvent("changed", false, false, null);
+                            var event = new CustomEvent("changed", {
+                                "bubbles":false,
+                                "cancelable":false,
+                                "detail":null
```
Comment (martinpitt) [inline pkg/kubernetes/scripts/graphs.js]: Note that the keys don't need to be quoted (and we usually don't), and there's a space missing after the `:`.

Comment (marusak) [reply pkg/kubernetes/scripts/graphs.js]: Thanks for pointing this out. Updated and (hopefully) remembered :)

**Pull request merged**